### PR TITLE
Implement multiple commands upload 

### PR DESCRIPTION
**Pull request by robinlin99**
- Refactor opcode/arg_1/arg_2 into RawRockblockCommand and RockblockCommand classes
- Update `CommandMonitor` and `RockblockControlTask` to support parsing of multiple commands in single transmission from ground station
- Update `test_command_monitor` and `test_report_acknowledge` test suites and tested implementation against aforementioned test suites
- Tested implementation using Rockblock Simulator and on EDU

### CODE REVIEW

File: src/Control Tasks/RockblockControlTask.cpp [hunk 1]
```diff
@@ -367,37 +367,67 @@ void RockblockControlTask::dispatch_process_command()
...
-        for (size_t o = 0; o < constants::rockblock::opcode_len; ++o) {
-            sfr::rockblock::opcode[o] = sfr::rockblock::serial.read();
-            if (sfr::rockblock::opcode[o] < 0x10)
-                Serial.print(0, HEX);
-            Serial.print(sfr::rockblock::opcode[o], HEX);
-        }
-        for (size_t a1 = 0; a1 < constants::rockblock::arg1_len; ++a1) {
-            sfr::rockblock::arg_1[a1] = sfr::rockblock::serial.read();
-            if (sfr::rockblock::arg_1[a1] < 0x10)
+        while (1) {
```
Comment (ljgreenhill) [inline src/Control Tasks/RockblockControlTask.cpp:370]: Infinite while?

**Review by zgk2003 — APPROVED** [review]
Looks good to me

File: src/Control Tasks/RockblockControlTask.cpp (hunk 1 above)
Comment (ljgreenhill) [reply src/Control Tasks/RockblockControlTask.cpp:370]: Even with the break it seems risky. Is there another way you could do this?

**Review by ljgreenhill — CHANGES_REQUESTED** [review]

File: src/Control Tasks/RockblockControlTask.cpp (hunk 1 above)
Comment (ljgreenhill) [reply src/Control Tasks/RockblockControlTask.cpp]: This change looks good to me!

**Review by ljgreenhill — APPROVED** [review]

**Pull request merged**